rules/library.bzl

```diff
@@ -243,7 +246,7 @@ def apple_library(name, library_tools = {}, export_private_headers = True, names
         else:
             fail("Unable to compile %s in apple_framework %s" % (f, name))
 
-    module_name = kwargs.pop("module_name", name)
+    module_name = _normalize_module_name(kwargs.pop("module_name", name))
```

Comment:
I think we should only normalize in the fallback case, i.e.

`kwargs.pop("module_name", _normalize_module_name(name))`

Reply:
Hmm... good point. I'll change it.